### ISSUE TITLE
Fix ManualDrive command so it's not using accumulated rotations when rotating to angle

### ISSUE THREAD

**Issue by celttechie** [FROG3160/2024-crescendo#198]
The navx gyro inherently accumulates rotations with it's getAngle function.  We need to make sure the angle returned is always constrained to -180 to 180 or -pi to pi so the controller doesn't "unwind" the robot trying to rotate back to a specific angle.

**Comment by celttechie** [comment]
This should probably be done AFTER #199 and use that method as the source for the current angle of the robot.

**Comment by celttechie** [comment]
This was already done.
